.github/workflows/test-k3s.yml

```diff
@@ -52,7 +58,15 @@ jobs:
         # NOTE: "PATH=$PATH" preserves the default user $PATH. This is needed to maintain the version of go installed
         #       in a previous step. This test run will use Zarf to create a K3s cluster, and a brand new cluster will be
         #       used for each test
-        run: sudo env "PATH=$PATH" CI=true APPLIANCE_MODE=true make test-e2e ARCH=amd64
+        run: |
+          sudo env "PATH=$PATH" CI=true APPLIANCE_MODE=true make test-e2e ARCH=amd64
+          sudo chmod 644 /tmp/zarf-*.log
```

Comment:
I know this is because of the `sudo` stuff, but it did take a second looking at it to figure out why this was here.  Not required to change, but at least for me something like this would be more clear:

```
sudo chown $USER:$(id -gn) /tmp/zarf-*.log
```

Reply:
That's fair, I think we could even just change user and that would be enough for it to be read.

Reply:
Yeah that should work as well